<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lenovo-PC" ignored="0" name="Default test(failed)(failed)" tests="2" failures="2" timestamp="31 Mar 2020 14:33:16 GMT" time="88.42" errors="0">
  <testcase name="login" time="32.253" classname="com.ivavsys.TestPOM.LoginTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.ivavsys.pom.EnterTimeTrack.Vaildate(EnterTimeTrack.java:23)
at com.ivavsys.TestPOM.LoginTest.login(LoginTest.java:29)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="31.52" classname="com.ivavsys.TestPOM.LoginTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.ivavsys.pom.EnterTimeTrack.Vaildate(EnterTimeTrack.java:23)
at com.ivavsys.TestPOM.LoginTest.login(LoginTest.java:29)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Default test(failed)(failed) -->
